ember/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-text-mask",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
Actually, I need to revert this change because it will be taken care of when I do `npm version major` before I publish to the npm registry.